storage/gcloud/aio/storage/storage.py

```diff
@@ -378,6 +387,10 @@ async def _download(self, bucket: str, object_name: str, *,
         s = AioSession(session) if session else self.session
         response = await s.get(url, headers=headers, params=params or {},
                                timeout=timeout)
+
+        if get_content:
+            return response.content
```

Comment:
This will only return a stream for `gcloud-aio` -- for gcloud-rest-storage, we'll need to make use of [iter_content()](https://docs.python-requests.org/en/latest/api/#requests.Response.iter_content), I think, and attach `stream=True` to the request.